Solution.py

```diff
@@ -0,0 +1,43 @@
+class Solution:
+    def EvenSum (list):
+        answer = 0
+        for i in range (len(list)):
+            if list[i] % 2 == 0:
+                answer += list[i]
+        return answer
+
+    def Common (list1, list2):
+        answer = []
+        for elem in list1:
+            if elem in list2:
+                answer.append(elem)
+        return answer
+
+    def Smallest2 (list : list):
```

Comment:
You can do this with list.sort and you can use slicing (list[-2]) to get the second smallest